signer/src/network/in_memory.rs

```diff
@@ -55,32 +57,21 @@ impl Default for Network {
 }
 
 impl super::MessageTransfer for MpmcBroadcaster {
-    type Error = Error;
-    async fn broadcast(&mut self, msg: super::Msg) -> Result<(), Self::Error> {
+    async fn broadcast(&mut self, msg: super::Msg) -> Result<(), Error> {
         self.recently_sent.push_back(msg.id());
-        self.sender.send(msg).map_err(|_| Error::Send)?;
+        self.sender.send(msg).unwrap();
```

Comment:
Are we okay with using unwraps here? Thinking if we ever did enable using the in-memory db at runtime (for regtest/testnet etc.)? I'm not against it, just that we have a decision that unwraps are OK for the in-mem DB.

Reply:
I don't know. I'm generally of the opinion unwraps aren't okay, but this is extremely  close to a testing utility. And unlike our in-memory db, I haven't used this stuff at all, so I had half a mind to remove it. 

But thinking about this some more, I'll switch back to using our normal pattern and later we can decide on what, if anything, to do with this module.

Reply:
Sounds good :+1: